includes/Modules/Ads.php

```diff
@@ -85,7 +93,64 @@ protected function setup_assets() {
 					),
 				)
 			),
+			new Script_Data(
+				'googlesitekit-ads-pax-config',
+				array(
+					'global'        => '_googlesitekitPAXConfig',
+					'data_callback' => function() {
+						if ( ! current_user_can( Permissions::VIEW_DASHBOARD ) ) {
+							return array();
+						}
+
+						return array(
+							'authAccess'      => array(
+								'oauthTokenAccess' => array(
+									'token' => (string) $this->authentication->get_oauth_client()->get_access_token(),
+								),
+							),
+							'locale'          => mb_substr( $this->context->get_locale( 'user' ), 0, 2 ),
+							'debuggingConfig' => array(
+								'env' => 'PROD',
+							),
+						);
+					},
+				)
+			),
 		);
+
+		if ( current_user_can( Permissions::VIEW_DASHBOARD ) && $this->is_connected() ) {
+			$assets[] = new Script(
+				'googlesitekit-ads-pax-integrator',
+				array(
+					'src'          => 'https://www.gstatic.com/pax/dev/pax_integrator.js',
+					'dependencies' => array(
+						'googlesitekit-ads-pax-config',
+						'googlesitekit-modules-data',
+					),
+				)
+			);
+		}
+
+		return $assets;
+	}
+
+	/**
+	 * Populates module data to pass to JS via _googlesitekitModulesData.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array $modules_data Inline modules data.
+	 * @return array Inline modules data.
+	 */
+	private function inline_data( $modules_data ) {
```

Comment:
How about `inline_modules_data` to be a bit more explicit about what the data is for outside of the doc block?